cli/loadCompletion.sh

```
@@ -0,0 +1,43 @@
+# Generated via opfab --completion
+# Added manually :  
+#    [[ $COMPREPLY ]] && return      
+#    _filedir
+# to solve issue with file completion see https://github.com/f/omelette/issues/4
+
+if type compdef &>/dev/null; then
+  _opfab_completion() {
+    compadd -- `opfab --compzsh --compgen "${CURRENT}" "${words[CURRENT-1]}" "${BUFFER}"`
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)